### PR TITLE
add KCD 2020 Berlin

### PR DESCRIPTION
**Pull request by cpanato**
Preliminary work on 2020-berlin branch. 
Need some checking and confirmation.

### CODE REVIEW

**Comment by KrissiWi** [comment]
Here comes my feedback and content

**General Feedback**

- If possible: Hide Program and Speakers in the menu until we have something

- If possible: Add CFP as menu point and redirect directly to sessionize [https://sessionize.com/kubernetes-community-days-berlin-2020/]

- Team: Add Pic Chris and Lexi & Caro  


**Text Content Landing Page**

Kubernetes Community Days are coming to Berlin! 

On May 26-27, the Kubernetes and cloud native community in Germany will gather at Alte Münze in Berlin. Join us for a two day technical conference loaded with exciting talks and plenty of opportunity to exchange and network. KCD Berlin are aimed at developers, operations people and all other IT professionals with an interest in cloud native technologies and are supported by the Cloud Native Computing Foundation. 

Button: “Register” with link to Eventbrite  

You have a Kubernetes story to tell? Great! Then don’t be shy and submit a proposal. Our CFP is open until March 15. 

Button “Submit CFP” [https://sessionize.com/kubernetes-community-days-berlin-2020/]

KCD is a community-organized and not for profit conference and would not be possible without our partners. Don’t miss the opportunity to be part of and support this community conference. Find our Partnership Proposal here. [Link to sponsorship prospectus]

**Text Content Program [if not possible to hide]**

Please have some patience. Currently, we are still collecting talks via our CFP [[https://sessionize.com/kubernetes-community-days-berlin-2020/]. Our agenda will be announced in early April. Follow us on Twitter [https://twitter.com/k8s_berlin] to stay up to date.


**Text Content Speakers [if not possible to hide]**

Please have some patience. Currently, we are still looking for speakers via our CFP [https://sessionize.com/kubernetes-community-days-berlin-2020/]. We will announce the first speakers as soon as possible. Follow us on Twitter [https://twitter.com/k8s_berlin] to stay up to date.

**Text Content Sponsor** 

KCD is a community-organized and not for profit conference that would not be possible without our partners. Don’t miss the opportunity to be part of and support this community conference. Find our Partnership Proposal here. [Link to sponsorship prospectus]

If you are interested in one of the available sponsorship packages or have any questions, please contact us at organizers-berlin@kubernetescommunitydays.org.

**Text Content Team**

No community conference without an awesome team of volunteers:-) If you want to get in touch please contact us at organizers-berlin@kubernetescommunitydays.org.   

**Text Venue** 

Alte Münze [https://alte-muenze-berlin.de/] is an old coin press building and lies in the very heart of Berlin,nearby the Spree river. We don’t want to promise too much, but Alte Münze will definitely give you the Berlin-vibe you are looking for:) 

Please have some patience. We’ll be adding directions soon.

**Comment by KrissiWi** [comment]
image

Can we add a real Button here? And if not just have it like "Submit CFP"

**Comment by KrissiWi** [comment]
image

Can we format this either as a real button or like the Submit CFP link

**Comment by KrissiWi** [comment]
Some minor changes:

- [ ] Bolden **Kubernetes Community Days are coming to Berlin and increase font size by 2 pt

- [ ] Speaker page: replace "call for participation" with "call for proposals" 

- [ ] Team page: add twitter links or remove the icon

- [ ] Team page: Make all the boxes have the same size

**Comment by iennae** [comment]
@KrissiWi  please be advised of the problem here https://github.com/cncf/kubernetes-community-days/issues/268 We'll try to get that resolved.

**Comment by Carolin519** [comment]
Hi Carlos,

here's our feedback regarding the website (printed in bold type and in square brackets):

Berlin Overview

 Submit CFP
- [ ] KCD is a community-organized and not for profit conference and would not be possible without our partners. Don’t miss the opportunity to be part of and support this community **[conference]** (replace 'conference' by 'event')...

Speakers
Call for Speakers

- [ ] The call for participation for **[a Kubernetes Community Days event]** (replace by 'the Kubernetes Community Days Berlin') is now open. 

- [ ] Follow the **[talk proposal submission process]** (link doesn't work yet) if you’d like to submit an idea and when ready to submit, head over to our CFP page.

Venue

- [ ] Alte Münze is an old coin press building and lies in the very heart of **[Berlin,nearby]** (comma is missing) the Spree river. We don’t want to promise too much, but Alte Münze will definitely give you the Berlin-vibe you are looking for:)

I hope that's understandable enough?

**Comment by Carolin519** [comment]
Hi Carlos,

can you also pls add Bill to the orga team on the website.

Here's the picture.
Foto Bill_Loodse

Thanks!

**Comment by Carolin519** [comment]
Hi Carlos.
One more thing for the website.
Can you also pls add our companies as partners on the website? Linking them with our pictures on the tab 'orga team' also would be great.
Thanks!

**Comment by Carolin519** [comment]
Here's our logo.
loodse4github

**Comment by cpanato** [comment]
@Carolin519 done

**Comment by tinerehbein** [comment]
Hey carlos,

pls add 

- SysEleven Logo on the Sponsor side (attached as png)
- Sponsor Prospectus (attached as PDF)
- Eventbrite Link https://www.eventbrite.com/e/kubernetes-community-days-berlin-2020-tickets-90095641489

[kcdberlin2020_sponsor_prospectus.pdf](https://github.com/cncf/kubernetes-community-days/files/4171280/kcdberlin2020_sponsor_prospectus.pdf)
SysEleven_XL_Logo_quer_RGB

thx tine

**Comment by cpanato** [comment]
@tinerehbein done 😄

**Comment by tinerehbein** [comment]
could you pls add under "Sponsors" the prospectus PDF?
anything else looks good.

**Comment by cpanato** [comment]
@tinerehbein tried, but this might be a limitation for the platform we are using

**Comment by KrissiWi** [comment]
Thanks Carlos. I just found one more little thing

We need to delete "Button: “Register” with link to Eventbrite" and move the register link up


image

**Comment by cpanato** [comment]
done @KrissiWi

**Comment by KrissiWi** [comment]
So, the only thing that is left to do is a way to add the sponsor PDF, right? 
Can't we just simply upload it in a public place and then add it via link here

image

**Comment by cpanato** [comment]
@KrissiWi checking the link was working, not sure why it is broken now :(
checking

**Comment by KrissiWi** [comment]
https://drive.google.com/file/d/1gfGZP0dWlhxSi9G6oHmkQeoswsE1M9PS/view?usp=sharing

**Comment by cpanato** [comment]
@KrissiWi Done :)

**Comment by KrissiWi** [comment]
Cool, thanks. So I ship the preview over for approval.

File: content/events/2020-berlin/_index.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+On May 26-27, the Kubernetes and cloud native community in Germany will gather at Alte Münze in Berlin. Join us for a two day technical conference loaded with exciting talks and plenty of opportunity to exchange and network. KCD Berlin are aimed at developers, operations people and all other IT professionals with an interest in cloud native technologies and are supported by the Cloud Native Computing Foundation.
+
+[Register](https://www.eventbrite.com/e/kubernetes-community-days-berlin-2020-tickets-90095641489)
+
+You have a Kubernetes story to tell? Great! Then don’t be shy and submit a proposal. Our CFP is open until March 15.
+
+[Submit CFP](https://sessionize.com/kubernetes-community-days-berlin-2020/)
+
+KCD is a community-organized and not for profit event and would not be possible without our partners. Don’t miss the opportunity to be part of and support this community event. Find our [Partnership Proposal](/img/2020-berlin/kcdberlin2020_sponsor_prospectus.pdf).
```
Comment (dankohn) [inline content/events/2020-berlin/_index.md]: Please change `Partnership Proposal` to `Partnership Prospectus`.

Comment (cpanato) [reply content/events/2020-berlin/_index.md]: done

**Comment by KTan1226** [comment]
Hi Berlin team- can you add a +1 from one of your organizers so we can merge? Thanks!

**Comment by LexiNadolski** [comment]
LGTM

**Pull request merged**